app/API/Client.php

```diff
@@ -40,10 +40,10 @@ public function get_events(): array {
 	/**
 	 * Get API event response object
 	 *
-	 * @param int $vdm_id
+	 * @param string $vdm_id
```

Comment:
Wil je dit aub niet dubbel doen als er al een typehint in de method staat?

Reply:
https://github.com/level-level/ll-voordemensen/pull/11/commits/ace305a95f964f147acd639e323300f2fb975685